src/isValidGlobalIdentifier.js

```diff
@@ -1,14 +1,15 @@
 const _ = require('lodash');
 const { definitions } = require('@identity.com/uca');
-const vcDefinitions = require('./creds/definitions');
-const claimDefinitions = require('./claim/definitions');
+const { schemaLoader } = require('./schemas/jsonSchema');
 
 const validUCAIdentifiers = _.map(definitions, d => d.identifier);
-const validClaimIdentifiers = _.map(claimDefinitions, d => d.identifier);
-const validVCIdentifiers = _.map(vcDefinitions, d => d.identifier);
+const validClaimIdentifiers = schemaLoader.validIdentifiers;
+const validVCIdentifiers = schemaLoader.validCredentialIdentifiers;
 const validPrefixes = ['claim', 'credential'];
 
-function isValidGlobalIdentifier(identifier) {
+async function isValidGlobalIdentifier(identifier) {
+  await schemaLoader.loadSchemaFromTitle(identifier);
```

Comment:
Add a comment about what is changing in the state, since you are awaiting but not really expecting a result

Reply:
Done